sections/futures/Trade/NextPrice.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import styled from 'styled-components';
+import { EXTERNAL_LINKS } from 'constants/links';
+
+// import Button from 'components/Button';
```

Comment:
Can remove a lot of comments here?

Reply:
@Tburm Yeah. Left those in because we will probably still need them when we add volatility control when placing next-price orders.